### ISSUE TITLE
Addon doesn't work anymore?

### ISSUE THREAD

**Issue by thibaultmol** [cpriest/SnapLinksPlus#317]
For some reason it seems like it doesn't actually open any of the links it detects anymore since yesterday.
Version 3.1.7
already tried reinstalling the addon

**Comment by remi-garcia** [comment]
Can you give more information on your web browser version and OS?

**Comment by thibaultmol** [comment]
Sorry yeah:
https://www.whatsmybrowser.org/b/PZ208

Firefox 84.0.1 (64-bit)

```
Operating System: Manjaro Linux
KDE Plasma Version: 5.20.4
KDE Frameworks Version: 5.77.0
Qt Version: 5.15.2
Kernel Version: 5.9.11-3-MANJARO
OS Type: 64-bit
Processors: 16 × AMD Ryzen 7 3700X 8-Core Processor
Memory: 31,3 GiB of RAM
Graphics Processor: GeForce GTX 1080/PCIe/SSE2
```

**Comment by remi-garcia** [comment]
Firefox was updated on the 22th of December to 84.0.1 for Arch based distribution - https://archlinux.org/packages/extra/x86_64/firefox/
On my Ubuntu Focal it is still 84.0 and everything is working fine. - https://launchpad.net/ubuntu/+source/firefox

Do you believe it is possible that it stop working right after you've updated Firefox?


Just in case for later : Firefox 84.0.1 [Release notes](https://www.mozilla.org/en-US/firefox/84.0.1/releasenotes/)

**Comment by thibaultmol** [comment]
... it's possible yeah. I don't exactly remember when it started happening, but it could be right.
 
You could try and launch the appimage version of the latest stable on your machine (don't have to install anything, just launch it) to see if you can replicate the issue
https://github.com/srevinsaju/Firefox-Appimage/releases/tag/stable

**Comment by remi-garcia** [comment]
There is no problem on my side with this appimage. Do you have this issue with this appimage? If not, which other add-ons do you have?

**Comment by thibaultmol** [comment]
The appimage still uses my current firefox config, I realized. So instead I tried a seperate/blank about:profiles
I still have the issue but I think I realized that I've been using it wrong?
I do get it to work properly if I let go of my modifier keys first and then let go of my mouse drag.
if I just keep my modifier keys pressed and let go of my mouse first instead, it doesn't open them.
Is this intended behaviour? (only recently switched to firefox and still had to jump between firefox and chrome. And the chrome extension I'm used to doesn't require you to let go of the modifier keys)
